src/core/models/quarters.ts

```diff
@@ -0,0 +1,51 @@
+import { Schema, model } from "mongoose";
+
+const QuarterSchema = new Schema({
+    
+    currentYear: { 
+        type: Number,
+        //i was thinking of using date for this field so this can be automated rather than us putting it in manually.
+        //type: Date,
+        min: 2024,
+    },
+
+    currentQuarter: {
```

Comment:
Should just be `quarter` since its value doesn't change once it has been created

Reply:
Same here